index.html

```diff
@@ -1,67 +1,219 @@
+                    <button class="button2">See Project</button>
+                </div>
+    
+                <div class="works7">
+                    <div class="image-back">
+                        <div class="professional"><h2 class="professional">Profesional Art Printing Data</h2></div>
+                        <div class="paragraphworks"><p class="paragraphworks">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                        <div>
+                            <ul class="boxes2">
+                                <li class="waks1"><a href="#"></a><img src="image/html2.png" alt="HTML"></li>
+                                <li class="waks2"><a href="#"></a><img src="image/bootstrap2.png" alt="Bootstrap"></li>
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button class="button2">See Project</button>
+                </div>
+            </div>
+        </div>
 
-        <div class="social-images">
-            <ul class="social">
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/github.png" alt="Github" class="im-age">
-                    </a>
-                </li>
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/linkedin.png" alt="LinkedIn" class="im-age">
-                    </a>
-                </li>
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/thumbs.png" alt="Thumbs" class="im-age">
-                    </a>
+        <div class="afterworks">
+            <div class="immediate-about">
+                <div class="aboutme"><h2 class="aboutme">About Me</h2></div>
+                <div class="aboutpara"><p class="aboutpara">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p></div>
+                <button class="aboutbutton"></button>
+            </div>
+            <div class="group2"><img src="image/corrected-group2.png" alt="Group2"></div>
+            <hr id="abbout">
+            <ul>
+                <li class="all-lists">
+                    <div class="language-container">
+                        <img src="image/diamond.png" alt="Diamond"> <h3 class="language">Language</h3>
+                    </div>
+                    <li id="similartex" class="javascript">JavaScript</li>
+                    <li class="ruby">Ruby</li>
+                    <li id="similartex" class="html">HTML</li>
+                    <li id="similartex" class="css">CSS</li>
                 </li>
```

Comment:
- [ ] please kindly fix the design for your technologies category it is currently not looking like the expected figma template, you are also required to use flexbox to structure the contents of this section kindly add that to your design :+1: 
- figma design
![f-s-s-s](https://user-images.githubusercontent.com/89055502/235335263-67913b7a-723a-4b68-acca-bba5ceb44263.png)
- your design 
![b-ss](https://user-images.githubusercontent.com/89055502/235335274-f5772e8c-5f9b-445c-a8b8-6ed618687ea9.png)